index.html

```diff
@@ -74,5 +76,179 @@ <h3>   I am a Software Developer </h3>
+          </h3>
+          <div class="project-img-btn">
+            <button type="button" class="btn2"><span>Html</span></button>
+            <button type="button" class="btn3"><span class="big-span">Bostrap</span></button>
+            <button type="button" class="btn4"><span>Ruby</span></button>
+            
+          </div>
+          <button type="button" class="project-btn2">See Project</button>
+       </section>
+       <section class="project-section-img">
+         <h2>Profesional Art 
+           Printing Data
+          </h2>
+          <h3 class="paragraph">
+            A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+             has been the industry's standard
+          </h3>
+          <div class="project-img-btn">
+            <button type="button" class="btn2"><span>Html</span></button>
+            <button type="button" class="btn3"><span class="big-span">Bostrap</span></button>
+            <button type="button" class="btn4"><span>Ruby</span></button>
+            
+          </div>
+          <button type="button" class="project-btn2">See Project</button>
+       </section>
+       <section class="project-section-img">
+         <h2>Profesional Art 
+           Printing Data
+          </h2>
+          <h3 class="paragraph">
+            A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+             has been the industry's standard
+          </h3>
+          <div class="project-img-btn">
+            <button type="button" class="btn2"><span>Html</span></button>
+            <button type="button" class="btn3"><span class="big-span">bootstrap</span></button>
+            <button type="button" class="btn4"><span>Ruby</span></button>
+            
+          </div>
+          <button type="button" class="project-btn2">See Project</button>
+       </section>
+       <section class="project-section-img">
+         <h2>Profesional Art 
+           Printing Data
+          </h2>
+          <h3 class="paragraph">
+            A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+             has been the industry's standard
+          </h3>
+          <div class="project-img-btn">
+            <button type="button" class="btn7">Html</button>
+            <button type="button" class="btn5">bootstrap</button>
+            <button type="button" class="btn6">Ruby</button>
+            
+          </div>
+          <button type="button" class="project-btn2">See Project</button>
+       </section>
+  </section>
+  <section class="about-me">
+    <h2 class="about-me-text">About me</h2>
+    <h3 class="about-me-paragraph">Hello I’m a software developer! I can help you build a product ,
+       feature or website Look through some of my work and experience!
+       If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.
+    </h3>
```

Comment:
- [ ] As in the previous comment I explained, for this description please also change the `h3` for a `p` tag